presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -439,6 +527,17 @@ private static ResourceEstimates parseResourceEstimate(MultivaluedMap<String, St
         return builder.build();
     }
 
+    /**
+     * Extracts the Presto_QUERY_REQUEST_METADATA header, parses escaped slashes and double quotes, than
```

Comment:
This information about parsing is not necessary.
Only keeps: "Extracts the Presto_QUERY_REQUEST_METADATA header"